index.js

```diff
@@ -30,6 +32,40 @@ else if(command.toLowerCase() === 'test') {
   commandStr.push("npm install -g jest")
   commandStr.push("jest --passWithNoTests ./test")
 }
+else if(command.toLowerCase() === 'auth') {
+  //generate jwt auth
+  const key = core.getInput('key')
+
+  const scopes = core.getInput('scopes')
+
+  const clientId = core.getInput('clientId')
+
+  const clientSecret = core.getInput('clientSecret')
+
+  const techAccId = core.getInput('technicalAccountId')
+
+  const imsOrgId = core.getInput('imsOrgId')
+
+  const imsConfig = {
+    client_id : clientId,
+    client_secret: clientSecret,
+    technical_account_id: techAccId,
+    ims_org_id: imsOrgId,
+    private_key: key.toString(),
+    meta_scopes: [
+      scopes
+    ]
+  }
+
+  getJwtToken(imsConfig)
+  .then(res => {
+    console.log('Generated auth token successfully')
+    core.exportVariable('AIO_ims_contexts_access__token_token', res)
```

Comment:
How about
```
core.setSecret(res)
```
to prevent it from being logged unmasked even accidentally. It needs to be tested if it works for subsequent commands in the github action.

Reply:
The env var set doesn't get logged as value but it masked and works for subsequent commands. Will try out core.setSecret and see if that can work for us as well.